src/_locales/en/messages.json

```diff
@@ -99,86 +99,86 @@
     "message": "Open Settings Page",
-    "description": "Alt text for the button that opens the Extensions Settings page"
+    "description": "Alternative text for the button that opens the Extensions Settings page."
   },
   "searchServer": {
     "message": "Search locations",
-    "description": "Placeholder for the Input field where users can search the list of servers"
+    "description": "Placeholder for the Input field where users can search the list of servers."
   },
   "titleSubscribeNow": {
-    "message": "Subscribe to Mozilla VPN"
+    "message": "Subscribe to Mozilla VPN."
   },
   "bodySubscribeNow": {
-    "message": "It looks like you're not subscribed to Mozilla VPN yet. To use this extension, please subscribe.",
+    "message": "It looks like you’re not subscribed to Mozilla VPN yet. To use this extension, please subscribe.",
     "description": "Body of an error message, shown if the user is trying to use the extension without a subscription."
   },
   "btnSubscribeNow": {
     "message": "Subscribe now",
-    "description": "Call to action button for users to subscribe to Mozilla VPN"
+    "description": "Call to action button for users to subscribe to Mozilla VPN."
   },
   "getHelp": {
     "message": "Get help",
-    "description": "Text of a link pointing to a relevant help page"
+    "description": "Text of a link pointing to a relevant help page."
   },
   "headerOpenMozillaVPN": {
     "message": "Open Mozilla VPN",
-    "description": "Headline"
+    "description": "Headline."
   },
   "headerSignedOut": {
-    "message": "Sign in to the Mozilla VPN app"
+    "message": "Sign in to the Mozilla VPN app."
   },
   "bodySignedOut": {
     "message": "You’re currently signed out of the Mozilla VPN app. To use the Mozilla VPN Firefox extension, open the app on your computer and sign in to your account."
   },
   "btnOpenVpn": {
-    "message": "Open Mozilla VPN"
+    "message": "Open Mozilla VPN."
   },
   "headerNeedsUpdate": {
     "message": "Update Mozilla VPN",
-    "description": "This is the headline of an error panel"
+    "description": "This is the headline of an error panel."
   },
   "bodyNeedsUpdate": {
     "message": "In order to use this extension, you must have the latest Mozilla VPN application installed.",
     "description": "This is the body copy for an error panel."
   },
   "headerInstallMsg": {
     "message": "Install Mozilla VPN",
-    "description": "Header if a VPN installation was not found"
+    "description": "Header if a VPN installation was not found."
   },
   "bodyOpenMsg": {
     "message": "To use the Mozilla VPN Firefox extension, please open the Mozilla VPN app first.",
-    "description": "Popup subtitle text"
+    "description": "Popup subtitle text."
   },
   "bodyInstallMsg": {
-    "message": "Note that Mozilla VPN is a paid subscription service."
+    "message": "To use this extension, first download and install the Mozilla VPN app on your computer."
```

Comment:
I believe this is the correct string based on Figma. Right now it's identical to the footer.

Reply:
Thank you!